mmv1/products/compute/PacketMirroring.yaml

```diff
@@ -143,7 +143,8 @@ properties:
     required: true
     properties:
       - name: 'subnetworks'
-        type: Array
+        type: !ruby/object:Api::Type::Array
```

Comment:
this can be reverted since we don't use ruby anymore for generation.

It's worth noting that this is a breaking change and won't be merged until the major release of 7.0.0